jac-cloud/jac_cloud/core/architype.py

```diff
@@ -387,15 +387,15 @@ def _pull(self) -> dict:
     def add_to_set(self, field: str, anchor: Anchor, remove: bool = False) -> None:
         """Add to set."""
         if field not in (add_to_set := self._add_to_set):
-            add_to_set[field] = {"$each": set()}
+            add_to_set[field] = {"$each": []}
 
-        ops: set = add_to_set[field]["$each"]
+        ops: list = add_to_set[field]["$each"]
```

Comment:
What's the thinking for switching this from set to a list?

Reply:
this is to retain the sequence of added objects